Automattic/wp-calypso
# 100 year plan: Add 100 Year checkout thank you UI

#Fixes Automattic/martech#2185

## Proposed Changes

* The checkout thank you page for the hundred year plan
* The `Start building` and ` Access premium support` buttons dynamically based on whether a user created an account within a day. If the user created the account before more than 1 day they will be shown `Start building` and otherwise the latter.

| Dekstop    | Mobile |
| -------- | ------- |
| <img width="1327" alt="image" src="https://github.com/Automattic/wp-calypso/assets/3422709/5525809f-d71a-4826-ac1c-b32c80039cf4"> |  <img width="486" alt="image" src="https://github.com/Automattic/wp-calypso/assets/3422709/5d36eab1-1f3f-4a38-8b9a-698e7233c998"> |

### Fresh Account

<img width="900" alt="image" src="https://github.com/Automattic/wp-calypso/assets/3422709/05e6d801-2781-4725-aad9-5700e82de31c">


## Temporary Testing Instructions



* Create a new free site
* Add free credits required for the 100 year plan from the store admin or **sandbox the store**
* Purchase a 100 year using the following link
    * `/checkout/<site-slug>/wp_bundle_hundred_year` or
    * completing the flow - `/setup/hundred-year-plan` 
* Make sure the following UI is visible

To trigger the fresh account based buttons. Create a fresh account (Created within 1 day) and retry the flow.

## Pre-merge Checklist
